packages/boxel-ui/addon/components/tooltip/index.gts

```diff
@@ -0,0 +1,63 @@
+import Component from '@glimmer/component';
+import { MiddlewareState } from '@floating-ui/dom';
+import { Velcro } from 'ember-velcro';
+import { on } from '@ember/modifier';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    placement?: MiddlewareState['placement'];
+    offset?: number;
+  };
+  Blocks: {
+    trigger: [];
+    content: [];
+  };
+}
+export default class Tooltip extends Component<Signature> {
+  @tracked isHoverOnHook = false;
```

Comment:
Would be good to decide on whether to use "hook" or "trigger" and consolidate so we don't have 2 namings for the same thing